client/knowledge_base_search/src/client/knowledge_base_search/node_modules/.bin/loose-envify

```diff
@@ -0,0 +1 @@
+../loose-envify/cli.js
```

Comment:
I don't know. I think it was a result of installing the dependencies for react-stickynode